### PR TITLE
Implemented Article Page routing and Article Layout

### PR DESCRIPTION
**Pull request by AmrTamer23**

### CODE REVIEW

File: src/app/articleGroup/[groupSlug]/article/[articleSlug]/page.tsx [hunk 1]
```diff
@@ -82,7 +82,7 @@ export default function Page({
       <main className="py-56 flex px-9">
         <aside className="flex flex-col pl-10 sticky top-10 h-full w-1/4">
           <span className="flex flex-col gap-3">
-            <h2>بواسطة مصطفي جمال مراجعة محمد سلطان</h2>
+            <h2 className="3xl:text-xl">بواسطة مصطفي جمال مراجعة محمد سلطان</h2>
```
Comment (Mahmoudgalalz) [inline src/app/articleGroup/[groupSlug]/article/[articleSlug]/page.tsx:85]: This is a bad for SEO

should be anything else, don't ever use h<> for anything, and leveling is so important

File: src/app/articleGroup/[groupSlug]/article/[articleSlug]/page.tsx [hunk 2]
```diff
@@ -125,25 +125,25 @@ export default function Page({
...
-        <aside className="flex gap-10 w-1/4 h-full  pr-5 sticky top-10">
+        <aside className="flex gap-10 w-1/4 h-full pr-5 sticky top-10">
           <Divider
             orientation="vertical"
             className="bg-divider w-0.5 h-11/12"
           />
           <div className="flex flex-col w-fit">
             <span className="flex flex-col gap-3">
-              <h6 className="text-sm">
+              <h6 className="text-sm 3xl:text-lg">
```
Comment (Mahmoudgalalz) [inline src/app/articleGroup/[groupSlug]/article/[articleSlug]/page.tsx:143]: The name of the article would be h6 ?! is that the title?

File: src/app/components/Article/ArticleFooter.tsx [hunk 3]
```diff
@@ -15,17 +15,19 @@ export function ArticleFooter({
         <AccordionItem
           key="1"
           aria-label="روابط مفيدة"
-          title={<h4 className="text-black text-xl"> روابط مفيدة</h4>}
+          title={
+            <h4 className="text-black text-xl 3xl:text-2xl"> روابط مفيدة</h4>
```
Comment (Mahmoudgalalz) [inline src/app/components/Article/ArticleFooter.tsx:19]: Cannot be h4 or any h

**Review by Mahmoudgalalz — CHANGES_REQUESTED** [review]
as I said, there's a lot of SEO things needs to be optimized

also we need to add meta,data here
specially 

so in the Layout make sure to include OG image same as the image in the article, title and description

File: src/app/articleGroup/[groupSlug]/article/[articleSlug]/page.tsx (hunk 2 above)
Comment (AmrTamer23) [reply src/app/articleGroup/[groupSlug]/article/[articleSlug]/page.tsx:143]: This is the main category's name, but I will change the h6 as it's bad for leveling

**Review by Mahmoudgalalz — APPROVED** [review]

**Pull request merged**